cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -354,10 +371,18 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
 
     String imageVersion = getString(properties, IMAGE_VERSION);
 
+    String gcpCmekKeyName = getString(properties, "gcpCmekKeyName");
+    String gcpCmekBucket = getString(properties, "gcpCmekBucket");
+
+    if ((gcpCmekKeyName == null) != (gcpCmekBucket == null)) {
```

Comment:
Why not just do a simple `||`?

`if (gcpKeyName == null || gcpBucket == null)`

Reply:
Since they are not tied together, I have removed the validation